samples/transform/README.md

```diff
@@ -0,0 +1,40 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates how to start transform jobs using your own transform script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](TODO).
+
+In order to follow this script, you must first create a transform script packaged in a Dockerfile that is [compatible with Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html). 
+
+You will also need a model in SageMaker for this sample. If you do not have one you must first create a [model](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-mkt-model-pkg-model.html)
+## Preparing the transform Script
```

Comment:
nit: Could we refer to it as `Batch Transform` rather than just `transform`? That's the official SageMaker title